.github/workflows/ci.yaml

```diff
@@ -70,20 +70,26 @@ jobs:
   integration-test:
     strategy:
       fail-fast: false
+      max-parallel: 6
       matrix:
         tox-environments:
           - backup-integration
           - charm-integration
           - database-relation-integration
           - db-relation-integration
-          - db-admin-relation-integration
           - ha-replication-integration
           - ha-self-healing-integration
           - ha-restore-cluster-integration
           - password-rotation-integration
           - plugins-integration
           - tls-integration
-    name: ${{ matrix.tox-environments }}
+        agent-versions:
+          - "2.9.43"  # renovate: latest juju 2
+          - "3.1.5"  # renovate: latest juju 3
+        include:
+          - tox-environments: db-admin-relation-integration
+            agent-versions: "2.9.43"  # renovate: latest juju 2
```

Comment:
Cannot deploy on juju 3

Reply:
This is the agent used for bootstraping. IIRC 2.9.44 got released while I was already working on this, but https://github.com/canonical/postgresql-operator/pull/190 will move this pin.